h/models/feature.py

```diff
@@ -7,15 +7,16 @@
 log = logging.getLogger(__name__)
 
 FEATURES = {
+    "client_display_names": "Render display names instead of user names in the client",
+    "client_user_profile": "Enable client-side user profile and preferences management",
     "embed_cachebuster": (
         "Cache-bust client entry point URL to prevent browser/CDN from "
         "using a cached version?"
     ),
-    "client_display_names": "Render display names instead of user names in the client",
+    "group_members": "Allow users to manage group members in new group forms",
+    "group_type": "Allow users to choose group type in new group forms",
     "pdf_custom_text_layer": "Use custom text layer in PDFs for improved text selection",
     "styled_highlight_clusters": "Style different clusters of highlights in the client",
-    "client_user_profile": "Enable client-side user profile and preferences management",
-    "group_type": "Allow users to choose group type in group creation form",
```

Comment:
While adding this flag I sorted the existing ones by name to make it more obvious where to slot new entries in. Looking at the rest of the code in this file it shouldn't matter what order the entries are in, and the admin UI already sorted flags for display, so there is no change there.

Reply:
:+1: